resources/src/mediawiki/mediawiki.jqueryMsg.js

```diff
@@ -124,7 +124,7 @@
 				return parser.parse( key, argsArray );
 			} catch ( e ) {
 				fallback = parser.settings.messages.get( key );
-				mw.log.warn( 'mediawiki.jqueryMsg: ' + key + ': ' + e.message );
+				console.log( 'mediawiki.jqueryMsg: ' + key + ': ' + e.message );
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this logging statement. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS2228)